.clang-format

```diff
@@ -1,15 +1,91 @@
-﻿---
+---
+AccessModifierOffset: -2
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: false
+AlignConsecutiveDeclarations: false
+AlignEscapedNewlinesLeft: true
+AlignOperands: true
+AlignTrailingComments: true
+AllowAllParametersOfDeclarationOnNextLine: false
+AllowShortBlocksOnASingleLine: false
+AllowShortCaseLabelsOnASingleLine: false
+AllowShortFunctionsOnASingleLine: Empty
+AllowShortIfStatementsOnASingleLine: false
+AllowShortLoopsOnASingleLine: false
+AlwaysBreakBeforeMultilineStrings: true
+AlwaysBreakTemplateDeclarations: true
 BasedOnStyle: WebKit
+BinPackArguments: false
+BinPackParameters: false
 BraceWrapping:
-  AfterClass: true
+  AfterClass: false
```

Comment:
We spent a bunch of time defining what we wanted for style, so I think it's best if we leave the decisions we had in place.

Reply:
> We spent a bunch of time defining what we wanted for style, so I think it's best if we leave the decisions we had in place.

FYI this `AfterClass: false` comes from BBP standards, but I have no objection since every project is free to define its own conventions.